setup.py

```diff
@@ -220,7 +219,14 @@
 #
 #   USE_PRIORITIZED_TEXT_FOR_LD
 #      Uses prioritized text form cmake/prioritized_text.txt for LD
+#
+#   BUILD_LIBTORCH_WHL
+#      Builds libtorch.so and its dependencies as a wheel
+#
+#   BUILD_PYTORCH_USING_LIBTORCH_WHL
```

Comment:
If we are not supporting this option ``BUILD_PYTORCH_USING_LIBTORCH_WHL`` yet should we be adding this code now ?